sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rule/BeamBasicAggregationRule.java

```diff
@@ -40,15 +50,21 @@
 
   public BeamBasicAggregationRule(
       Class<? extends Aggregate> aggregateClass, RelBuilderFactory relBuilderFactory) {
-    super(operand(aggregateClass, operand(TableScan.class, any())), relBuilderFactory, null);
+    super(operand(aggregateClass, operand(AbstractRelNode.class, any())), relBuilderFactory, null);
```

Comment:
Looking at examples of this in Calcite, I think `RelNode` is preferable to `AbstractRelNode`.

Reply:
You are correct, updated match condition to use RelNode instead